Review request: Designing Courses and Writing section updates (#25)
RedHatTraining/curriculum-onboarding: content/upskilling/02.adoc

---
title: Design Meetings
weight: 52
tags:
 - design
personas:

>   - Curriculum developer

Author: I will update the personas. I tried to do that but obviously missed many. It wasn't a focus for this first run.

---

== What is a Design Meeting and Why do I Care?

The design of a course is one of the more important pre-development tasks course development teams and the Instructional designer undertakes.

Course design speaks to the following elements

Objective wording and alignment with topics and exercises (specifically at the section level).
Content depth based on course level (a 100-level course has different design needs than a 400-level course).
The flow of content from chapter-to-chapter. Determine what content is repeated throughout the course and determine the depth of detail for each section.
Graphic treatments and screenshot opportunities are considered, noted.
Write modularly, as much as is appropriate.

> 

Author: Updated this section.

> === When and How do We Conduct a Design Meeting?

Author: I will consider this change. They don't bother me in a less formal environment.

In the current model (2021) we are developing content in phases. In a traditional model, the design meeting is held just before course development begins but because we develop in phases, we have two opportunities to influence design. First, let’s define the phases.

> **Phase I (MVP)** is just Guided Exercises or End-of-chapter Labs and Open content with light or no narrative. The idea is to create an MVP to gather feedback from customers, internal partners, and stakeholders. Content Services is not required to review anything at this point but provides assistance, as needed. The exercises, in this phase, can be developed with minimal step instructions but typically will include grading scripts.

Reviewer: Make *Phase I (MVP)* a heading.
```suggestion
==== Phase I (MVP)
This phase includes guided exercises and end-of-chapter labs and "OPEN" content with light or no narrative. The idea is to create an MVP to gather feedback from customers, internal partners, and stakeholders. Content Services is not required to review anything at this point but provides assistance, as needed. The exercises in this phase can be developed with minimal step instructions but typically will include grading scripts.
```
afaik MVP doesn't include grading scripts.

> **Phase II (General Availability)** is when chapter content, including narrative and labs, is built out fully to be ready for release. Where Phase I content is minimal, this phase is where everything is fleshed out so the learner has all the information needed to be successful in the course.

Reviewer: ```suggestion
==== Phase II (General Availability)
This phase is when the balance of the content, including narrative and labs, is fully developed and reviewed ready for release. Where Phase I content is minimal, this phase is where everything is fleshed out so the learner has all the information needed to be successful in the course.
```

> 

Reviewer: Phase III? I know there is no design meeting for this phase but we should let them know that it exists.

> === Design Meetings in Phases

Reviewer: Design meetings are not held in phases. Perhaps you want something like "Design Meeting Opportunities" or "Scheduling Design Meetings" or similar?

Author: Updated.

> There are two opportunities to conduct design meetings.
> Once before the MVP Phase and again before Phase II begins.
> They break down in the following manner.

Reviewer: ```suggestion
Design meetings are scheduled at two junctures: The first is held before Phase I begins and the second before Phase II begins. The agenda of each is described below.
```

=== Phase I Design Meeting

> The ID, Curriculum Architect, and development team meet to discuss the approach to creating the Guided Exercises and/or Labs.

Reviewer: ```suggestion
This meeting addresses how to approach creating the guided exercises and labs.
```

> Using the course outline, which may be in one of these formats, Detailed Course Outline or Timing document, listing the chapters, sections, and exercise titles, we review the following

Reviewer: ```suggestion
This meeting uses the Detailed Course Outline (DCO) to review the following topics:
```
I **know** the "DCO" comes in a variety of formats but I think it's pointless to list all the different guises under which it may appear. If we just say DCO then ppl get the idea.

Author: Updated.

The relevance of the GE - does it still fit in the course flow?
Are there complex exercises or labs that can be broken down?

> Which GE’s cannot be developed due to classroom environment constraints or issues?

Reviewer: Don't write "GE's" or "CA's" or whatever when you're talking about plurals.

> Which GE’s cannot be developed due to classroom environment constraints or issues?

Reviewer: ```suggestion
Which GEs cannot be developed due to classroom issues?
```

=== Phase II Design Meeting

> All the same players from Phase I meet but the objective of this design meeting is to define design for each chapter in multiple, shorter, focused meetings, focusing on the following

Reviewer: ```suggestion
The objective of this meeting is to define the design of each chapter in multiple, shorter, focused meetings, focusing on the following:
```
I'm not sure that this makes sense. The phase 2 design meeting is actually a series of shorter meetings? If so, you need to spell that out better.

> Content flow from chapter-to-chapter - based on the course outline, do the topics make sense in the order they’re in?

Reviewer: ```suggestion
Content flow from chapter to chapter. Based on the course outline, do the topics make sense in their current order?
```

Within each chapter, do the objectives make sense? Are they measurable and observable?

> Ask questions as a team about all aspects of the chapter being reviewed, to give the Architect an opportunity to clarify.

Reviewer: ```suggestion
Ask questions as a team about all aspects of the chapter being reviewed, to give the architect an opportunity to clarify.
```

> Make suggestions for modifying content, adding or pulling and

Reviewer: Incomplete sentence.
